### PR TITLE
Feat/verification api verification status

### PR DESCRIPTION
**Pull request by Dovchik**
Add [Verification Status](https://developers.sinch.com/docs/verification/api-reference/verification/tag/Verification-status/) calls. Further tests with doppelganger.

### CODE REVIEW

File: src/Sinch/Verification/VerificationMethod.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿using Sinch.Core;
+
+namespace Sinch.Verification
+{
+    /// <summary>
+    ///     The method of the verification request.
+    /// </summary>
+    /// <param name="Value"></param>
+    public record VerificationMethod(string Value) : EnumRecord(Value)
```
Comment (asein-sinch) [inline src/Sinch/Verification/VerificationMethod.cs:9]: Maybe this record can be renamed to `VerificationStatusMethod` to match the documentation

Comment (asein-sinch) [inline src/Sinch/Verification/VerificationMethod.cs:9]: Btw, is there a reason why this one is a `record` whereas the `VerificationTypeInternal` (which could be renamed to `VerificationMethod`) is a `class` ?

Comment (Dovchik) [reply src/Sinch/Verification/VerificationMethod.cs:9]: Internal is for use of as constants in code. Record is specifically for this param: https://developers.sinch.com/docs/verification/api-reference/verification/tag/Verification-status/#tag/Verification-status/operation/VerificationStatusByIdentity!in=path&path=method&t=request

Comment (asein-sinch) [reply src/Sinch/Verification/VerificationMethod.cs:9]: Up to you, but as it is also described as `method` in the Verifications API, I was thinking about mentioning the VerificationStatus API name to clearly make the distinction.

**Review by asein-sinch — APPROVED** [review]

**Pull request merged**